homeassistant/components/wyoming/assist_satellite.py

```diff
@@ -41,19 +43,47 @@
 _PING_TIMEOUT: Final = 5
 _PING_SEND_DELAY: Final = 2
 _PIPELINE_FINISH_TIMEOUT: Final = 1
-_CONVERSATION_TIMEOUT_SEC: Final = 5 * 60  # 5 minutes
+_STOP_CHUNK: Final = b""
 
 # Wyoming stage -> Assist stage
-_STAGES: dict[PipelineStage, assist_pipeline.PipelineStage] = {
+_ASSIST_STAGES: dict[PipelineStage, assist_pipeline.PipelineStage] = {
     PipelineStage.WAKE: assist_pipeline.PipelineStage.WAKE_WORD,
     PipelineStage.ASR: assist_pipeline.PipelineStage.STT,
     PipelineStage.HANDLE: assist_pipeline.PipelineStage.INTENT,
     PipelineStage.TTS: assist_pipeline.PipelineStage.TTS,
 }
+_WYOMING_STAGES: dict[assist_pipeline.PipelineStage, PipelineStage] = {
+    assist_pipeline.PipelineStage.WAKE_WORD: PipelineStage.WAKE,
+    assist_pipeline.PipelineStage.STT: PipelineStage.ASR,
+    assist_pipeline.PipelineStage.INTENT: PipelineStage.HANDLE,
+    assist_pipeline.PipelineStage.TTS: PipelineStage.TTS,
+}
+
 
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up VoIP Assist satellite entity."""
+    domain_data: DomainDataItem = hass.data[DOMAIN][config_entry.entry_id]
+    assert domain_data.satellite_device is not None
+
+    async_add_entities(
+        [
+            WyomingSatellite(
+                hass, config_entry, domain_data.service, domain_data.satellite_device
+            )
+        ]
+    )
 
-class WyomingSatellite:
-    """Remove voice satellite running the Wyoming protocol."""
+
+class WyomingSatellite(WyomingEntity, assist_satellite.AssistSatelliteEntity):
```

Comment:
Please open a standalone PR to add the `WyomingSatellite` implementation.  The trigger PR has to be layered on top.